go.mod

```diff
@@ -23,13 +23,13 @@ require (
 	github.com/open-policy-agent/opa v0.35.0
 	github.com/pkg/errors v0.9.1
 	github.com/secure-systems-lab/go-securesystemslib v0.3.0
-	github.com/sigstore/fulcio v0.1.2-0.20211207184413-f4746cc4ff3d
-	github.com/sigstore/rekor v0.3.1-0.20211211150321-b8eca1b71e0b
-	github.com/sigstore/sigstore v1.0.2-0.20211223185235-75bc699f921d
+	github.com/sigstore/fulcio v0.1.2-0.20220114150912-86a2036f9bc7
+	github.com/sigstore/rekor v0.4.1-0.20220114213500-23f583409af3
+	github.com/sigstore/sigstore v1.1.1-0.20220114020210-b1a36a4771f1
```

Comment:
same here: should we pin for the tagged release or do we want to use the head of main?

Reply:
Updating to head is fine!